third_party/musl/src/exit/atexit.c

```diff
@@ -54,12 +54,14 @@ void __funcs_on_exit()
 	}
 }
 
+#if !defined(ADDRESS_SANITIZER)
 void __cxa_finalize(void *dso)
 {
 #ifdef STARBOARD
   __funcs_on_exit();
 #endif  // STARBOARD
 }
+#endif
```

Comment:
should be #endif // !defined(ADDRESS_SANITIZER)  fot his one and others

Reply:
Done